docs/cli.md

```diff
@@ -0,0 +1,63 @@
+# Goose CLI Commands
```

Comment:
I assume this part is generated? and if I make a change to the CLI args, PRs will fail until this is updated, right?

Reply:
nope this one is not generated